src/FondOfSpryker/Zed/CompanyUserCompanyAssigner/Business/Model/CompanyRoleInterface.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace FondOfSpryker\Zed\CompanyUserCompanyAssigner\Business\Model;
+
+use Generated\Shared\Transfer\CompanyUserTransfer;
+
+interface CompanyRoleInterface
+{
+    /**
+     * @param \Generated\Shared\Transfer\CompanyUserTransfer $companyUserTransfer
+     *
+     * @return void
+     */
+    public function updateNonManufacturerUsersCompanyRole(
+        CompanyUserTransfer $companyUserTransfer
+    ): void;
+
+    /**
+     * @param \Generated\Shared\Transfer\CompanyUserTransfer $companyUserTransfer
+     *
+     * @return array<int, array<string, mixed>>
+     */
+    public function findCompanyUsersWithDiffCompanyRolesAsManufacturer(
```

Comment:
```suggestion
    public function findWithOldCompanyRoles(
```

Reply:
@daniel-rose here is used the same naming as first suggested **_findCompanyUsersWithOldCompanyRoles_**